Neos.Flow/Classes/Security/Authentication/Token/UsernamePassword.php

```diff
@@ -62,27 +68,27 @@ public function updateCredentials(ActionRequest $actionRequest)
             return;
         }
 
-        $credentials = $this->extractCredentialsFromRequest($actionRequest);
+        $username = $this->getUsername();
```

Comment:
`$this->actionRequest` is not set before calling those two methods. What about passing the ActionRequest as argument instead of relying on the property?